### PR TITLE
Repayments

### PR DESCRIPTION
**Pull request by axshani**
implementation of Repayments
https://developers.unit.co/repayments

### CODE REVIEW

File: unit/models/repayment.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    def to_dict(self) -> Dict:
+        parameters = {"page[limit]": self.limit, "page[offset]": self.offset}
+
+        if self.account_id:
+            parameters["filter[accountId]"] = self.account_id
+
+        if self.credit_account_id:
+            parameters["filter[creditAccountId]"] = self.credit_account_id
+
+        if self.customer_id:
```
Comment (GermanSmoliar) [inline unit/models/repayment.py:97]: Lines 97 - 98 are the same as 100 - 101

**Review by GermanSmoliar — CHANGES_REQUESTED** [review]

File: unit/models/repayment.py (hunk 1 above)
Comment (axshani) [reply unit/models/repayment.py:97]: when you are right you are right

File: unit/models/repayment.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        self.transaction_summary_override = transaction_summary_override
+        self.tags = tags
+        self.idempotency_key = idempotency_key
+        self.relationships = relationships
+
+    def to_json_api(self) -> Dict:
+        return super().to_payload("bookRepayment", self.relationships)
+
+
+class CreateAchRepaymentRequest(UnitRequest):
```
Comment (GermanSmoliar) [inline unit/models/repayment.py:48]: sameDay is missing here

File: unit/models/repayment.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        self.id = _id
+        self.type = _type
+        self.attributes = extract_attributes(["amount", "status", "tags"], attributes)
+        attrs = {"createdAt": date_utils.to_datetime(attributes["createdAt"]),
+                 "updatedAt": date_utils.to_datetime(attributes["updatedAt"])}
+        self.attributes.update(attrs)
+        self.relationships = relationships
+
+
+class BookRepaymentDTO(BaseRepayment):
```
Comment (GermanSmoliar) [inline unit/models/repayment.py:18]: Maybe I am wrong but tags are inside of attributes, and I they are missing

**Review by GermanSmoliar — CHANGES_REQUESTED** [review]

File: unit/models/repayment.py (hunk 3 above)
Comment (axshani) [reply unit/models/repayment.py:18]: `self.attributes = extract_attributes(["amount", "status", "tags"], attributes)`

**Comment by GermanSmoliar** [comment]
@AsafPolakovski , approved from my side. Can you go over?

**Review by AsafPolakovski — APPROVED** [review]

**Pull request merged**